### PR TITLE
Fix some styling and typo's in README's

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
Also fix some styling and typo's

### CODE REVIEW

**Comment by StevenLooman** [comment]
I'd rather not include lists of properties in the README files. These are bound to get out of date. It would be better to easily get the properties and/or automatically get these to the wiki.

File: magik-lint/README.md [hunk 1]
```diff
@@ -62,30 +62,62 @@ The configuration file is located as follows, in order:
...
+| todo-comment | forbidden-comment-words | TODO,FIXME,HACK,NOTE,TEMP,XXX |
+| unused-variable | check-parameters | false |
+| variable-count | max-variable-count | 8 |
+| variable-declaration-usage-distance | max-distance | 5 |
+| variable-declaration-usage-distance | ignore-constants | true |
+| variable-naming | min-length | 3 |
+| variable-naming | max-length | 32 |
+| variable-naming | whitelist | x,y,z |
+| warned-call | warned-calls | write(),sw:write(),remex(),sw:remex(),remove_exemplar(),sw:remove_exemplar() |
+
```
Comment (sebastiaanspeck) [inline magik-lint/README.md]:
```suggestion
```

File: magik-typed-lint/README.md [hunk 2]
```diff
@@ -36,30 +36,38 @@ The configuration file is located as follows, in order:
...
-line-length.line-length=120
+module-required-for-global.always-loaded-modules=ace_core,rwo_core,super_dd,sw_core_magik_sessions
 ```
 
+### Settings per check
+
+| Check name | Setting name | Default value |
+|------------|--------------|---------------|
+| module-required-for-global | always-loaded-modules | See [ModuleRequiredForGlobalTypedCheck](https://github.com/StevenLooman/magik-tools/blob/main/magik-typed-checks/src/main/java/nl/ramsolutions/sw/magik/typedchecks/checks/ModuleRequiredForGlobalTypedCheck.java#L51) |
+
```
Comment (sebastiaanspeck) [inline magik-typed-lint/README.md]:
```suggestion
```

**Comment by sebastiaanspeck** [comment]
> I'd rather not include lists of properties in the README files. These are bound to get out of date. It would be better to easily get the properties and/or automatically get these to the wiki.

Will look for the possibilities in a new PR.

**Comment by sebastiaanspeck** [comment]
> > I'd rather not include lists of properties in the README files. These are bound to get out of date. It would be better to easily get the properties and/or automatically get these to the wiki.
> 
> Will look for the possibilities in a new PR.

Something like this? https://github.com/sebastiaanspeck/magik-tools/wiki/Checks-Options

**Comment by StevenLooman** [comment]
Thanks!

**Pull request merged**